examples/rust/tracecon/src/bpf/tracecon.bpf.c

```diff
@@ -0,0 +1,134 @@
+#include "vmlinux.h"
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_tracing.h>
+
+#define memcpy(dest, src, n) __builtin_memcpy((dest), (src), (n))
+
+// Copied from: include/netdb.h
+struct addrinfo
+{
+	int ai_flags;         		/* Input flags.  */
+	int ai_family;        		/* Protocol family for socket.  */
+	int ai_socktype;      		/* Socket type.  */
+	int ai_protocol;      		/* Protocol for socket.  */
+	u32 ai_addrlen;       		/* Length of socket address.  */ // CHANGED from socklen_t
+	struct sockaddr *ai_addr; 	/* Socket address for socket.  */
+	char *ai_canonname;       	/* Canonical name for service location.  */
+	struct addrinfo *ai_next; 	/* Pointer to next in list.  */
+};
+
+struct lookup_T {
```

Comment:
the _T suffix is a POSIX thing -- `lookup` should be fine alone
```suggestion
struct lookup {
```